k8s/demo/load-generator.yaml

```
@@ -0,0 +1,60 @@
+    - |
+      echo "🔥 CPU Stress Test Pod Ready!"
+      echo "To stress test the service and trigger HPA scaling:"
+      echo "stress --cpu 1 --timeout 300s"
+      sleep 3600
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "50m"
+      limits:
+        memory: "128Mi"
+        cpu: "200m" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)